pegasus/sites.v3/code.org/public/css/common.css

```diff
@@ -403,6 +403,17 @@ button {
   margin-top: 10px;
 }
 
+a.link-button {
+  display: inline-block;
+  color: white;
+  background-color: #ffa400;
+  border-radius: 4px;
+  padding: 6px 12px;
+  font-size: 14px;
+  font-family: 'Gotham 4r', sans-serif;
+  text-decoration: none;
+}
```

Comment:
@mgc1194 just updated the styles 🎉 

Here's how it looks:

<img width="1010" alt="Screenshot 2023-02-01 at 2 32 05 PM" src="https://user-images.githubusercontent.com/9256643/216179489-d3b24130-11f8-465e-9f20-16aff6abdf5d.png">